front-end-tools/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# This dockerfile is meant to be run from the **root of the repository**.
+
+ARG build_image=node:16-slim
+
+FROM ${build_image} AS build
+
+RUN yarn set version 3.2.0
+
+# Copy front end files
+WORKDIR /app
+COPY ./front-end-tools ./
+
+WORKDIR /app/front-end-tools
+RUN yarn install && yarn build
```

Comment:
We should have a `.dockerignore` file and have docker ignore `node_modules` otherwise your local build of packages might end in the image. Will also speed up the docker build time.

Reply:
Instead of `.dockerignore`, I specified the 5 files that should be copied into the docker image.